File: 240611/1-9 개수 세기/count-one-to-nine.py
N = int(input())
n_lst = list(map(int, input().split()))

n_lst_cntr = {n: 0 for n in range(1, 10)}
for n in n_lst:
    if n in n_lst_cntr:
        n_lst_cntr[n] += 1

for n, cnts in n_lst_cntr.items():
    print(cnts)

File: 251116/1-9 개수 세기/count-one-to-nine.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() 
{
    int N;
    cin >> N;

    vector<int> n_lst(N);
    for (int i = 0; i < N; i++) 
        cin >> n_lst[i];

    vector<int> n_lst_cntr(10, 0);

    for (int n : n_lst) 
    {
        if (1 <= n && n <= 9)
            n_lst_cntr[n]++;        
    }

    for (int i = 1; i <= 9; i++)
        cout << n_lst_cntr[i] << '\n';

    return 0;
}
